### ISSUE TITLE
General discussion

### ISSUE THREAD

**Issue by captainrobbo** [thameshareandhounds/thameshareandhounds.github.io#10]
Github doesn't provide a discussion or mailing list feature, but it just occurred to me that we can achieve this by leaving an issue open and replying to it.   

Anyone got opinions on which of the side-menu items really need to be there at the top level, and which could be demoted?

**Comment by captainrobbo** [comment]
Ed, I found that the embedded calendar you added was preventing the page shrinking nicely to mobile sizes, so have temporarily removed it.   Is that calendar something the club actively uses, or just an experiment?  We could probably give it a page of its own, or make it resize nicely within some other component

**Comment by ecatmur** [comment]
Just an experiment - we don't have anything similar on the current website. Some kind of list of upcoming fixtures & training runs might be good though.

**Comment by captainrobbo** [comment]
Yes, I agree.

There is a feature/trick in Jeykll whereby you can keep JSON files in
version control as a "data source" and then we could hook them up to a nice
Javascript calendar, or indeed filter them and pull them onto separate
pages.  Or Javascript could then show the upcoming ones and lose all the
past ones automatically.     I have no strong feelings except that if key
data is spread across half a dozen hosted systems, it will be harder to
keep track of.

**Comment by njg234908** [comment]
How about this for a simple & clean theme:

http://jekyllthemes.org/themes/pithy/

**Comment by ecatmur** [comment]
Oh, definitely - in fact that calendar is generated from yaml in _data/fixtures/2015-16.yml, I was just using Google Calendar as a renderer.

**Comment by captainrobbo** [comment]
I agree, that is very clean.  Under the hood the markup is much saner too.
I'd quibble that you lose a third of the screen to the top navigation (if
I've got to the page I want, e.g. results/fixtures, I'd like to see as much
as possible) buut I am sure the top bar could be slimmed.  I also wonder if
it supports any kind of drop down menus, because we do have quite a few
other pages, and we want to be only a lick or two away from them.

Unfortunately we spend days headbanging on three different themes before
stabilising on the current one and getting about half way into dealing with
all the issues.   It will drive Llewellyn nuts if we switch again.   If you
use a well publicised framework like Bootstrap or Material, you can do
anything and it's documented, but the markup sucks as it contains a hundred
things you don't need; and if you use something slim and specific, you
might have to extend it yourself.

Suggestion:  it's all driven by that layout file "default.html".  How about
trying an alternate one with Pithy?  We could then switch quite easily, and
indeed try others.

Go to go now - off to Watford tonight!
- Andy

**Comment by njg234908** [comment]
Ok, i purely picked that one out based on look and feel so perhaps we can
tweak the existing MDL theme rather than throwing out what we've done?

If we could make the left hand menu hidden by default on all devices,
accessible via the 'three horizontal lines' button, with the most common
items permanently available across the top menu. See the android.com
http://www.getmdl.io/templates/android-dot-com/index.html example on the
MDL site for roughly what I mean.

I will try and hack this if I can with the existing code!

**Comment by captainrobbo** [comment]
That should be easy enough.   Or we can just ditch or comment out the left
navigation.

In your view, which are the "three most common" in case I get time to play
today?   I may not though, have a lot of stuff to get done by the weekend.
- Andy

**Comment by njg234908** [comment]
About / News / Fixtures
